src/table/table.scss

```
@@ -367,6 +412,42 @@
     flex-basis: $iui-l * 2;
   }
 
+  &.iui-sticky {
```

I think we should be more specific here than just `iui-sticky` as we're working on avoiding class name collisions between components.

```suggestion
  &.iui-cell-sticky {
```

---

But is it needed for a modifier?

---

nothing is "needed" but it's better to avoid collisions just in case

---

Changed